setup.cfg

```diff
@@ -1,3 +1,8 @@
+[mypy]
+
+[mypy-smart_open.*]
+ignore_missing_imports = True
```

Comment:
Could be mistaken, but it seems to pass linting locally without this?

Reply:
I'm assuming this file is replaced by `pyproject.toml` in the next PR, can I leave until it's removed?

Reply:
Good point!  Yep, think it will be removed.